src/test/java/bio/terra/common/SynapseUtils.java

```diff
@@ -0,0 +1,77 @@
+package bio.terra.common;
+
+import bio.terra.model.BillingProfileModel;
+import bio.terra.service.filedata.azure.AzureSynapsePdao;
+import bio.terra.service.filedata.azure.blobstore.AzureBlobStorePdao;
+import bio.terra.service.filedata.azure.util.BlobContainerClientFactory;
+import bio.terra.service.resourcemanagement.azure.AzureStorageAccountResource;
+import com.microsoft.sqlserver.jdbc.SQLServerDataSource;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.stringtemplate.v4.ST;
+
+@Component
+public class SynapseUtils {
+  private static final Logger logger = LoggerFactory.getLogger(SynapseUtils.class);
+  private static final String readFromParquetFile =
+      "SELECT *\n"
+          + "FROM OPENROWSET(\n"
+          + "    BULK '<parquetFilePath>',\n"
+          + "    DATA_SOURCE = '<dataSourceName>',\n"
+          + "    FORMAT = 'parquet') as rows";
+
+  @Autowired AzureSynapsePdao azureSynapsePdao;
+  @Autowired AzureBlobStorePdao azureBlobStorePdao;
+
+  public List<String> readParquetFileStringColumn(
+      String parquetFilePath, String dataSourceName, String columnName, boolean expectSuccess) {
```

Comment:
When would `expectSuccess` be false?

Reply:
In cleanup, I run this again after we delete the parquet files to confirm that they are no longer around. I wanted this condition so that we don't litter the logs w/ error messages that we expect to occur.